src/main/java/org/prebid/server/geolocation/MaxMindGeoLocationService.java

```diff
@@ -68,7 +69,7 @@
             return Future.succeededFuture(GeoInfo.builder()
                     .vendor(VENDOR)
                     .continent(getCity(inetAddress).getContinent().getCode().toLowerCase())
```

Comment:
1. Extract `getCity(inetAddress)` and `getLocation(inetAddress)` to separate variables.
2. Refactor fetching fields of `GeoInfo` with the using of `resolveXXX` methods which should checks NPE at each getting stage.